src/pages/dashboard/index.jsx

```diff
@@ -244,7 +244,7 @@ function Navigation() {
 
   return (
     <>
-      <SideNavigation header={navHeader} items={navItems} activeHref="#/" onFollow={e => onFollowHandler(e)} />
+      <CommonNavigation items={navItems} activeHref="#/" onFollow={e => onFollowHandler(e)} />
```

Comment:
SideNavigation is now unused, we can remove that import

Reply:
done